<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PD2 Effective Damage Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="class_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Class:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="class_cbox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ama</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nec</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>pal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dru</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ass</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="skill_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>16</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Skill:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="skill_cbox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>8</number>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>351</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="skills_grid"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>540</y>
      <width>661</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="damage_grid" columnstretch="0" columnminimumwidth="0"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>381</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="elemental_grid"/>
   </widget>
   <widget class="QLabel" name="mon_type_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Monster Type: </string>
    </property>
   </widget>
   <widget class="QComboBox" name="mon_type_cbox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>21</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Maps</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ubers</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Key Holders</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hell Bosses</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hell Mobs</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Superuniques</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="tier_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="tier_cbox">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="mon_label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Monster: </string>
    </property>
   </widget>
   <widget class="QComboBox" name="mon_cbox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="area_label">
    <property name="geometry">
     <rect>
      <x>496</x>
      <y>63</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="area_cbox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mon_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>22</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Manual resistance entry</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="skill_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Manual skill damage entry</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hover over underlined text for more information, or see Info tab</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>556</x>
      <y>320</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>** SKILLS AND SYNERGIES UPDATED FOR S9 **</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>** Maps: resistance information S9; tier organization S7 **</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="info_summons"/>
    <addaction name="info_carryover"/>
    <addaction name="info_break"/>
    <addaction name="info_su"/>
    <addaction name="info_lod"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="info_summons">
   <property name="text">
    <string>Summons and Sentries</string>
   </property>
  </action>
  <action name="info_dmg">
   <property name="text">
    <string>Damage Calculations</string>
   </property>
  </action>
  <action name="info_su">
   <property name="text">
    <string>Superuniques</string>
   </property>
  </action>
  <action name="info_lod">
   <property name="text">
    <string>LoD/D2R</string>
   </property>
  </action>
  <action name="info_break">
   <property name="text">
    <string>Break</string>
   </property>
  </action>
  <action name="info_carryover">
   <property name="text">
    <string>Carryover Damage</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
